docs/docs/scanner/misconfiguration/policy/exceptions.md

```diff
@@ -92,7 +92,7 @@ You can get the package names in the [trivy-policies repository][trivy-policies]
 
 For more details, see [an example][rule-example].
 
-[ns-example]: https://github.com/aquasecurity/trivy/tree/{{ git.commit }}/examples/misconf/namespace-exception
-[rule-example]: https://github.com/aquasecurity/trivy/tree/{{ git.commit }}/examples/misconf/rule-exception
+[ns-example]:https://github.com/aquasecurity/trivy/tree/main/integration/testdata/fixtures/repo/namespace-exception
+[rule-example]:https://github.com/aquasecurity/trivy/tree/main/integration/testdata/fixtures/repo/rule-exception
```

Comment:
`{{ git.commit }}` are placeholder tags that get replaced upon generation with the appropriate tag.